sklearn/tests/test_multiclass.py

```diff
@@ -971,3 +972,19 @@ def test_support_missing_values(MultiClassClassifier):
     lr = make_pipeline(SimpleImputer(), LogisticRegression(random_state=rng))
 
     MultiClassClassifier(lr).fit(X, y).score(X, y)
+
+
+def test_constant_int_target():
+    """Check that constant y target does not raise.
+
+    Non-regression test for #21869
+    """
+    X = np.ones((10, 2))
+    y = np.zeros((10, 1), dtype=np.int32)
```

Comment:
Maybe it would be worth add the case `y = np.ones((10, 1), dtype=np.int32)` to this test?

Reply:
Added requested tests in [`a0dea78` (#21871)](https://github.com/scikit-learn/scikit-learn/pull/21871/commits/a0dea7898a65ab828cce9b2473d0ebb5ea9cf465)